Common/cpp/Tools/JSISerializer.h

```diff
@@ -2,7 +2,103 @@
 
 #include <jsi/jsi.h>
 #include <string>
+#include <vector>
 
 using namespace facebook;
 
+namespace {
+const std::vector<std::string> SUPPORTED_ERROR_TYPES = {
```

Comment:
do we need to expose those vectors? Unnamed namespace in a header makes those objects avaialble in all files it is included so no reason for unnamed namespace here